NewSpring/ApollosProject
# Updates to Core 0.8.3

## DESCRIPTION

### What does this PR do, or why is it needed?

Includes new assets updates with splash screens

### How do I test this PR?

boot up app on both OS's and click around to make sure our new features work.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] Closes [DEV-3717](https://newspring.atlassian.net/browse/DEV-3717)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
